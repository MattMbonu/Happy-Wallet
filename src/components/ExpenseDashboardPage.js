import React from "react";
import ConnectedExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const ExpenseDashboardPage = () => (
  <div>
    <p>this is my expenses Dashboard</p>
    <ExpenseListFilters />
    <ConnectedExpenseList />
  </div>
);

export default ExpenseDashboardPage;
