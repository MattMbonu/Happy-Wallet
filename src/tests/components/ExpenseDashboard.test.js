import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";

test("renders expense dashboard page", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
