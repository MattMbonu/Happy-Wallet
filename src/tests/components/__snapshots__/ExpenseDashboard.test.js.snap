// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders expense dashboard page 1`] = `
<div>
  <p>
    this is my expenses Dashboard
  </p>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
